decidim-core/spec/validators/password_validator_spec.rb

```diff
@@ -140,6 +141,28 @@
       end
     end
 
+    describe "parts of email domain included in password" do
+      context "when less than 4 character parts" do
+        let(:email) { "john.doe@1.lvh.me" }
+        let(:value) { "Summer1Snowlvhme" }
+
+        it "ignores domain validation" do
+          expect(validator).to be(true)
+          expect(record.errors[attribute]).to be_empty
+        end
+      end
+
+      context "when 4 or more character parts" do
+        let(:email) { "john.doe@decidim.com" }
```

Comment:
I'm a fan of using example.org/example.com for tests and documentation. See https://www.iana.org/domains/reserved
```suggestion
        let(:email) { "john.doe@example.org" }
```